apps/thunderhub/docker-compose.yml

```diff
@@ -19,6 +18,7 @@ services:
       ACCOUNT_CONFIG_PATH: "/data/thubConfig.yaml"
       YML_ENV_1: "$LND_IP:$LND_GRPC_PORT"
       TOR_PROXY_SERVER: "socks://$TOR_PROXY_IP:$TOR_PROXY_PORT"
+      MEMPOOL_URL: "http://${DEVICE_HOSTNAME}.local:$APP_MEMPOOL_PORT"
```

Comment:
@lukechilds Can you chime in here and construct the proper URL for pointing at the umbrel hosted mempool? Not sure I did it correctly.

Reply:
I changed it from `"http://${DEVICE_HOSTNAME}.local:$APP_MEMPOOL_PORT"` to `"http://${DEVICE_HOSTNAME}:$APP_MEMPOOL_PORT"`